src/malli/swagger.cljc

```diff
@@ -67,3 +72,85 @@
                                  ::json-schema/definitions definitions
                                  ::json-schema/transform -transform})]
      (cond-> (-transform ?schema options) (seq @definitions) (assoc :definitions @definitions)))))
+
+(defmulti extract-parameter (fn [in _] in))
+
+(defmethod extract-parameter :body [_ schema]
+  (let [swagger-schema (transform schema {:in :body, :type :parameter})]
+    [{:in          "body"
+      :name        (:title swagger-schema "body")
+      :description (:description swagger-schema "")
+      :required    (not= :maybe (m/type schema))
+      :schema      swagger-schema}]))
+
+(defmethod extract-parameter :default [in schema]
+  (let [{:keys [properties required definitions]} (transform schema {:in in, :type :parameter})]
+    (println "\nextract-parameter definitions:"
```

Comment:
this broke reitit build, fixed.

Reply:
https://github.com/metosin/malli/pull/878/files?w=1

Reply:
Oof sorry for not noticing that last println!

Reply:
no prob, the build broke with clojure 1.10 because of not requiring the `clojure.pprint`.